src/extlibs/hrvo/agent.cpp

```diff
@@ -145,3 +149,9 @@ void Agent::setRadius(float radius)
 {
     radius_ = radius;
 }
+
+void Agent::updateRadiusFromVelocity()
+{
+    // Linearly increase radius based on the current agent velocity
+    radius_ = min_radius_ + max_radius_inflation_ * (velocity_.length() / max_speed_);
```

Comment:
Do we have a limit of how large `max_radius_inflation` could be? We could check that `1 <= max_radius_inflation < ___`, if outside the bounds, default to a reasonable value?

Reply:
I think in this case we're trusting the user to pass in the correct value. The main limitation here is that `max_radius_inflation_` should be a positive number. I think in scenarios where we might be worried about the incorrect values being passed in, we could use something like `CHECK(value) << "Error message to print if value is false"` in the setter or the constructor.